### PR TITLE
Add-linters

### PR DESCRIPTION
**Pull request by Jeanbulambo**
In this pull request, I implemented the following elements:

- Create an `index.html` file
- Create an `style.css` file
- Add Linters
- etc

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,97 @@
...
+![gh actions checks](../assets/images/gh-actions-html-css-checks.png)
+
+Click on the `Details` link of each action to see the full output and the errors that need to be fixed:
+
+![gh actions failing checks](../assets/images/gh-actions-html-css-failing-checks.png)
+
+## Set-up linters in your local env
+
+**Note**: The `npm` package manager is going to create a `node_modules` directory to install all of your dependencies. You shouldn't commit that directory. To avoid that, you can create a [`.gitignore`](https://git-scm.com/docs/gitignore) file and add `node_modules` to it:
+
```
Comment (Kandy-Peter) [inline README.md:25]: - [ ] As requested, In this project, you should use the[README template](https://github.com/microverseinc/readme-template/blob/master/README.md) provided by Microverse.

**Review by Kandy-Peter — CHANGES_REQUESTED** [review]
Hi @tresorsawasawa 

Good job so far! I know you have worked hard on making this milestone. There is an issue that you still need to work on to go to the next project but you are almost there!

almost-there-almost


## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- You  are following GitHub actions :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
## Required Changes ♻️

- [ ] As required, your PR should a meaningful description of what you did in this part of the project. Please, add some more descriptions for the correct Usage of Github flow.
```
In this pull request, I implemented the following elements:

- Create an `index.html` file
- Add Linters
- etc

```

- [x] Kindly check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Jeanbulambo** [comment]
Please, for required changes, i don't understand very well. what should I do exactly.  Am I going to do again the project from the beginning then push so I may add description or I can do it from github?

**Comment by Kandy-Peter** [comment]
You can do it from GitHub, just click on the right top dots then edit and save.

**Comment by Jeanbulambo** [comment]
Please see a screenshot of what I edited according to the requirement and your orientation
pull.

Should I submit it like this?

**Review by Jeanbulambo — COMMENTED** [review]
comment pull request I did and update README file using Microverse Template

File: README.md [hunk 2]
```diff
@@ -1 +1,65 @@
-# Hello-World3
\ No newline at end of file
+# Project Name
+
+> Description the project.
```
Comment (omarsalem7) [inline README.md:3]: - [ ] You are make great work, but you should update the readme template according to your project. Please, add the project name `Hello world` for example.
- [ ] Also add a description of the project for example `This is my first project hello world in Microverse`

File: README.md [hunk 3]
```diff
@@ -1 +1,65 @@
...
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
```
Comment (omarsalem7) [inline README.md:34]: - [ ] Kindly remove this and add the instructions for how to clone your repository

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @Jeanbulambo , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

Almost near

 ## Highlights
✔️ All linter checks pass
✔️ Descriptive PR

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- Nothing to mention.
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 4]
```diff
@@ -1 +1,65 @@
...
+
+### Copy the URL for the repository.
+
+### Open Terminal.
+
+### Change the current working directory to the location where you want the cloned directory.
+
+### Type git clone, and then paste the URL you copied earlier.
+
+### Press Enter to create your local clone.
```
Comment (muneebulrehman) [inline README.md:35]: Optional.
Only use headings for a section heading. The rest of the content can be simple text or a list as you may desire.

File: README.md [hunk 5]
```diff
@@ -1 +1,65 @@
...
+
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@gJeanbulambo4](https://github.com/Jeanbulambo)
+- Twitter: [@Jeanbulambo](https://twitter.com/Jeanbulambo)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
```
Comment (muneebulrehman) [inline README.md:44]: Optional.
On any project, on which you are working alone, You can remove Authors heading and simply use **Author** as a heading followed with your social handles.

**Review by muneebulrehman — APPROVED** [review]
Hi @Jeanbulambo , 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🥳
Congratulations! 🎉 

# Status: Project Approved.

## Highlights
- Good follow up with previous reviewers
- All linter checks passing

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**